bundles/org.openhab.binding.comfoair/src/main/java/org/openhab/binding/comfoair/internal/ComfoAirHandler.java

```diff
@@ -140,6 +152,8 @@ public void dispose() {
             comfoAirConnector.close();
         }
 
+        updateState(ACTIVATE_CHANNEL_ID, OnOffType.OFF);
```

Comment:
I added these (also in the `connect()` method) to ensure that it is reflected in the item/channel state that the control is handed back to the external controller by `comfoAirConnector.close()`
https://github.com/openhab/openhab-addons/blob/1bb3b54f8d8d0bfef826f10ea776d1b3e4e55c73/bundles/org.openhab.binding.comfoair/src/main/java/org/openhab/binding/comfoair/internal/ComfoAirSerialConnector.java#L114
I admit that I'm not quite sure if this is necessary (if openHAB is shut down probably not, but what if the thing is just deactivated?). Thus, I'm fine with removing it here, if you think it's overkill.

Reply:
In the case where the thing is deactivated it *might* be ok. But since you can't tell the difference between thing deactivation and openhab shutdown it is better to play it safe and always treat it as a shutdown. So please remove any channel updates during disposal. If you need to send some commands to the device as part of shutdown then that is ok.